lms/static/js/components/header/models/header.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * A generic header model.
+ */
+;(function (define) {
+'use strict';
+define(['backbone'], function (Backbone) {
+    var HeaderModel = Backbone.Model.extend({
```

Comment:
Question/suggestion: I'm curious about naming conventions and how they reflect the extent to which we're trying to be generic. To me, this name implies a generic model to represent any header at all. Is that what we want? If so, this should live outside the teams app. Otherwise, I recommend we name it `TeamsHeaderModel`.